tests/BusinessTest.php

```diff
@@ -175,7 +175,7 @@ public function testClosestBeforeCustomTimezone()
         $date = new \DateTime('2015-05-25 10:00', new \DateTimeZone('Pacific/Tahiti'));
         $closest = $business->closest($date, Business::CLOSEST_LAST);
 
-        $this->assertEquals('2015-05-25 17:00', $closest->format('Y-m-d H:i'));
+        $this->assertEquals('2015-05-25 14:00', $closest->format('Y-m-d H:i'));
```

Comment:
I think you were mislead by the method name. We expect the last business time to be returned which is `17:00`.